wasm/wasm_source/src/vp_user.rs

```diff
@@ -134,14 +134,7 @@ fn validate_tx(
             KeyType::PoS => validate_pos_changes(ctx, &addr, key, &valid_sig)?,
             KeyType::PgfSteward(address) => address != &addr || *valid_sig,
             KeyType::GovernanceVote(voter) => voter != &addr || *valid_sig,
-            KeyType::Vp(owner) => {
-                let has_post: bool = ctx.has_key_post(key)?;
-                if owner == &addr {
-                    has_post && *valid_sig
-                } else {
-                    true
-                }
-            }
+            KeyType::Vp(owner) => owner != &addr,
```

Comment:
@cwgoes a note on this. This will prevent any vp update even if we wanted to allow it later. To do so we'd need to hard fork and modify this line so that vp updates are enabled. We's also need to modify all the accounts `?` storage key to point to a different vp hash. At the same time we can't allow vp changes in here (even though we removed them from the tx) cause it will be possible to to change VP via a sequence of governance proposals